admin_ui/src/components/EditRowForm.vue

```diff
@@ -60,31 +46,31 @@ export default Vue.extend({
         }
     },
     computed: {
-        schema() {
+        schema(): any {
             return this.$store.state.schema
         },
-        selectedRow() {
+        selectedRow(): any {
```

Comment:
We need to try and avoid `any` is possible. Does `object` work here?

Reply:
I agree with you that we need to try and avoid `any` if is possible but unfortunately `object` does not work.